openverse_catalog/dags/common/requester.py

```diff
@@ -79,7 +79,9 @@ def get_response_json(self, endpoint, retries=0, query_params=None, **kwargs):
                 logger.warning(f"Could not get response_json.\n{e}")
                 response_json = None
 
-        if response_json is None or response_json.get("error") is not None:
+        if response_json is None or (
```

Comment:
Victoria returns a list instead of an object in `json()` call, so we cannot use `response_json.get('error')` with it. `response_json` is a list.

Reply:
@stacimc , do you think this is a good solution here for cases where `response_json` is a list, not a dictionary?

Reply:
I guess the alternative would be to override `get_response_json` on `VictoriaDataIngester` to use `delayed_requester.get` instead of `.get_response_json`, like the original provider script -- in which case we miss out on error handling and retries. I don't love that the API is returning json without a top level object, but I think this is the best solution on our end. At any rate we're actually doing more error handling now than we were before! :)